src/main/java/core/basesyntax/vehicles/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.vehicles;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Bulldozer is working");
```

Comment:
The message in the `doWork` method should be more informative and indicate that it is a Bulldozer that is working. Consider specifying the machine type in the message, for example: 'Bulldozer started its work.'